be/src/vec/core/block.h

```diff
@@ -95,8 +95,9 @@ class Block {
     Block(Block&& block) = default;
     Block& operator=(Block&& other) = default;
 
+    /// Reserve memory for internal containers
     void reserve(size_t count);
-    // Make sure the nammes is useless when use block
+    /// Clear all column names and name index mappings in the block
```

Comment:
like these, we'd better make people know `index_by_name` and all `data`'s name will be clear.